### ISSUE TITLE
NPCs perform multiple attacks in one update

### ISSUE THREAD

**Issue by ifyousayso** [LightsHope/server#1214]
This doesn't happen every time, but under certain circumstances, some NPCs (all?) attack 2-3 times at once:
Bloodfury Roguefeather (4023)

It's an ancient bug where some NPCs may or may not have been "fixed" already.

**Comment by Cabro** [comment]
It's due to the stacking of the thrashing attacks while the NPCs target is out of range.

**Comment by ohganot** [comment]
It's because that NPC has Thrash ability, which grants 2 extra attacks.

**Comment by Cabro** [comment]
Yes but why is it stacking and queuing when target is out of range?

**Comment by ifyousayso** [comment]
Okay, then Thrash is broken because sometimes it does only two attacks (two hits and no misses).

.. but I'll post other NPCs here, as I find them, that don't have that (or similar) ability.
